hare/eligibility/oracle.go

```diff
@@ -245,6 +253,7 @@ func (o *Oracle) actives(layer types.LayerID) (map[string]struct{}, error) {
 		o.Error("Could not calculate hare active set size: no contextually valid blocks",
 			log.Uint64("layer_id", uint64(layer)), log.Uint64("epoch_id", uint64(layer.GetEpoch(o.layersPerEpoch))),
 			log.Uint64("safe_layer_id", uint64(sl)), log.Uint64("safe_epoch_id", uint64(safeEp)))
+		o.lock.Unlock()
```

Comment:
move above log

Reply:
It is true that it can be done before but as a convention, I prefer to stick the unlock next to the return.